content/develop/get-started/vector-database.md

```diff
@@ -87,123 +94,122 @@ This quick start guide also uses the **bikes** dataset. Here is an example docum
+### 2. Store the demo data in Redis
+Now iterate over the `bikes`  array to store the data as [JSON]({{< relref "/develop/data-types/json/" >}}) documents in Redis by using the [JSON.SET]({{< baseurl >}}/commands/json.set//) command. The below code uses a [pipeline]({{< relref "/develop/use/pipelining" >}}) to minimize the network round-trip times:
 
 {{< clients-example search_vss load_data />}}
 
-You can now retrieve a specific value from one of the JSON documents in Redis using a [JSONPath](https://goessner.net/articles/JsonPath/) expression:
+Once loaded, you can retrieve a specific attributes from one of the JSON documents in Redis using a [JSONPath](https://goessner.net/articles/JsonPath/) expression:
 
 {{< clients-example search_vss get />}}
 
-### 3. Select a machine-learning model
+### 3. Select a text embedding model
 
-This quick start guide uses a pre-trained [MS MARCO](https://microsoft.github.io/msmarco/) model. They are widely used in search engines, chatbots, and other AI applications.  
+[HuggingFace](https://huggingface.co) has a large catalog of text embedding models that are locally servable through the `SentenceTransformers` framework. Here we use the [MS MARCO](https://microsoft.github.io/msmarco/) model which is widely used in search engines, chatbots, and other AI applications.
 
 ```python
 from sentence_transformers import SentenceTransformer
 
 embedder = SentenceTransformer('msmarco-distilbert-base-v4')
 ```
 
-### 4. Create the vector embeddings
-In the next step, you must iterate over all the Redis keys with the prefix `bikes:`:
+### 4. Generate text embeddings
+Iterate over all the Redis keys with the prefix `bikes:`:
 
 {{< clients-example search_vss get_keys />}}
 
-Use the keys as a parameter to the [JSON.MGET]({{< baseurl >}}/commands/json.mget) command, along with the JSONPath expression `$.description` to collect the descriptions in a list. Then, pass the list to the `encode` method to get a list of vectorized embeddings:
+Use the keys as input to the [JSON.MGET]({{< baseurl >}}/commands/json.mget//) command, along with the `$.description` field, to collect the descriptions in a list. Then, pass the list of descriptions to the `.encode()` method:
 
 {{< clients-example search_vss generate_embeddings />}}
 
-You now need to add the vectorized descriptions to the JSON documents in Redis using the [JSON.SET]({{< baseurl >}}/commands/json.set) command. The following command inserts a new field in each of the documents under the JSONPath `$.description_embeddings`. Once again, you'll do this using a pipeline:
+Insert the vectorized descriptions to the bike documents in Redis using the [JSON.SET]({{< baseurl >}}/commands/json.set//) command. The following command inserts a new field in each of the documents under the JSONPath `$.description_embeddings`. Once again, do this using a pipeline to avoid unnecessary network round-trips:
```

Comment:
```suggestion
Insert the vectorized descriptions to the bike documents in Redis using the [JSON.SET]({{< baseurl >}}/commands/json.set) command. The following command inserts a new field into each of the documents under the JSONPath `$.description_embeddings`. Once again, do this using a pipeline to avoid unnecessary network round-trips:
```